manage/encrypted-volumes.md

```diff
@@ -63,3 +63,47 @@ Volume successfully created: 374663852714325215
 ```
 
 __Important: Make sure secret `key1` exists in the secret endpoint__
+
+## Encrypted Shared Volumes
+
+Encrypted shared volume allows access from multiple nodes to the same
+encrypted volume.
+
+Shared flag can be set while creating the encrypted volume using `--shared`
+It can also be enabled or disabled during run-time using `--shared on/off`.
+Volume must be in detached to state to enable shared flag on during run-time.
+
+All nodes in the cluster must be authenticated to access secret store for
+the encryption keys.
+Both cluster wide and per volume secrets are supported. For example, using
+cluster wide secret key;
+
+```
+#pxctl volume create --shared --secure --size 10 encrypted_volume
```

Comment:
space after #

Reply:
done

Reply:
you forgot to add space  after the # and before pxctl